lib/dirac_mobius.cpp

```diff
@@ -81,26 +69,26 @@ namespace quda {
     ApplyDslash5(out, in, in, mass, m5, b_5, c_5, 0.0, dagger, DSLASH5_MOBIUS);
 
     long long Ls = in.X(4);
-    long long bulk = (Ls-2)*(in.Volume()/Ls);
-    long long wall = 2*in.Volume()/Ls;
-    flops += 48LL*(long long)in.Volume() + 96LL*bulk + 120LL*wall;
+    long long bulk = (Ls - 2) * (in.Volume() / Ls);
+    long long wall = 2 * in.Volume() / Ls;
+    flops += 48LL * (long long)in.Volume() + 96LL * bulk + 120LL * wall;
   }
 
   // Modification for the 4D preconditioned Mobius domain wall operator
-  void DiracMobius::Dslash4Xpay(ColorSpinorField &out, const ColorSpinorField &in,
-				const QudaParity parity, const ColorSpinorField &x, const double &k) const
+  void DiracMobius::Dslash4Xpay(ColorSpinorField &out, const ColorSpinorField &in, const QudaParity parity,
+                                const ColorSpinorField &x, const double &k) const
   {
     checkDWF(in, out);
     checkParitySpinor(in, out);
     checkSpinorAlias(in, out);
 
     ApplyDomainWall4D(out, in, *gauge, k, m5, b_5, c_5, x, parity, dagger, commDim, profile);
 
-    flops += (1320LL+48LL)*(long long)in.Volume();
+    flops += (1320LL + 48LL) * (long long)in.Volume();
```

Comment:
Why is this an explicit sum? I understand if it's useful to distinguish the two contributions, but a quick comment on it would be instructive.